include/swift/SILOptimizer/Analysis/IVAnalysis.h

```diff
@@ -26,7 +26,6 @@ namespace swift {
 class IVInfo : public SCCVisitor<IVInfo> {
 public:
   typedef llvm::SmallVectorImpl<ValueBase *> SCCType;
-  friend class SCCVisitor;
```

Comment:
I get an error: “class template has already been declared as a non-class template”

I think the fix is to change this to `friend calss SCCVisitor<IVInfo>` which compiles in VS. I'm running a full build now to make sure